valid-anagram/sunjae95.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * @description
+ * time complexity: O(N)
+ * space complexity: O(N)
+ *
+ * brainstorming:
+ * 1. hash table value compare to count
+ *
+ * strategy:
+ * string change to hash table
+ */
+var isAnagram = function (s, t) {
+  if (s.length !== t.length) return false;
+
+  let answer = true;
+  const tableS = convertHashTable(s);
+  const tableT = convertHashTable(t);
+
+  tableS.forEach((_, key) => {
+    if (tableT.get(key) && tableT.get(key) === tableS.get(key)) return;
+    answer = false;
+  });
```

Comment:
nit: 요렇게 하면 `forEach()` 함수에 인자로 넘기는 콜백 함수의 첫 번째 인자를 낭비하지 않을 수 있지 않을까요? 그리고 어차피 `tableT.get(key)`가 `undefined`를 반환할텐데, 굳이 첫 번째 조건이 필요할까 생각이 들었어요.

```suggestion
  tableS.forEach((value, key) => {
    if (tableT.get(key) === value) return;
    answer = false;
  });
```

Reply:
오! 좋은 방법인데요?!
리뷰 감사합니다 :)